CoreFoundation/NumberDate.subproj/CFTimeZone.c

```diff
@@ -70,7 +72,7 @@ struct tzhead {
 
 #include <time.h>
 
-#if !TARGET_OS_WIN32
+#if !TARGET_OS_WIN32 && !TARGET_OS_ANDROID
```

Comment:
Hmm, I think that this can now be rewritten to `TARGET_OS_LINUX || TARGET_OS_BSD`.

Reply:
Oh, bleh, I missed that.  Yeah, the negation makes sense I suppose.